client/src/components/CopyToClipboardButton.tsx

```diff
@@ -7,35 +6,9 @@ type Props = {
 };
 
 export default function CopyToClipboardButton({ value }: Props) {
-  const textAreaRef = useRef(null);
+  const copyToClipboard = () => {
```

Comment:
I would recommend to use https://github.com/streamich/react-use/blob/master/docs/useCopyToClipboard.md as we already have this library in our app.

Reply:
updated